crds/AzureApplicationGatewayLoadDistributionPolicy.yaml

```diff
@@ -0,0 +1,57 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: azureapplicationgatewayloaddistributionpolicies.appgw.ingress.azure.io
+spec:
+  group: appgw.ingress.azure.io
+  scope: Namespaced
+  names:
+    kind: AzureApplicationGatewayLoadDistributionPolicy
+    plural: azureapplicationgatewayloaddistributionpolicies
+    singular: azureapplicationgatewayloaddistributionpolicy
+    shortNames:
+      - agldp
+  versions:
+    - name: v1
+      served: true
+      storage: true
+      schema:
+        openAPIV3Schema:
+          type: object
+          properties:
+            spec:
+              type: object
+              properties:
+                targets:
+                  description: "An array of services that contains information for how the service should be used for load balancing and active passive support in Application Gateway"
```

Comment:
`An array of services that contains information for how the service should be used for load balancing in Application Gateway`